app/services/induction/amend_participant_cohort.rb

```diff
@@ -102,10 +111,22 @@ def initialize(*)
       @target_cohort_start_year = (@target_cohort_start_year || @schedule&.cohort_start_year).to_i
     end
 
+    def back_to_payments_frozen_cohort?
+      participant_profile&.cohort_changed_after_payments_frozen? && target_cohort&.payments_frozen?
```

Comment:
In @ethax-ross pr there are a few extra check for declarations existing in one and not the other, maybe worth duplicating the logic here as well?